osier/technology.py

```diff
@@ -220,6 +239,14 @@ def __init__(self,
     def __repr__(self) -> str:
         return (f"{self.technology_name}: {self.capacity}")
 
+    def __eq__(self, tech) -> bool:
+        """Test technology equality"""
+        if ((self.technology_name == tech.technology_name)
+                and (self.capacity == tech.capacity)):
+            return True
+        else:
+            return False
+
```

Comment:
Is this a placeholder? Or is this just a generic function for now? I can imagine that you may want to check more than just the name and capacity.

Reply:
It's not a placeholder, I needed to implement this to test the `synchronize_units` function. The test ensures that the technologies are still the same, even though the units are different. In general, it's not a particularly useful feature because there are lots of ways to define equality. I chose one that is simple and necessary.